### DISCUSSION TITLE
gm_notification  and Chrome

### DISCUSSION THREAD

**Discussion by nathanrod** [violentmonkey/violentmonkey#1405] [Q&A]
Hi:

I have a very simple script that basically displays a notification when a page is loaded but seems not working with Google Chrome. I verified it is working with Firefox but i'm unable to make it work with Chrome..

Things i tried:

Tried to use tampermonkey 
Verify that the site is whitelisted in my Chrome notification settings
Disable all my active addons

Example:

```

// ==UserScript==
// @name        _Notification test
// @match       *://YOUR_SERVER.COM/YOUR_PATH/*
// @match       https://stackoverflow.com/questions/51769201/*
// @grant       GM_notification
// @require     http://code.jquery.com/jquery-1.12.4.min.js
// ==/UserScript==

GM_notification ( {
    title: 'foo', text: '42', image: 'https://i.stack.imgur.com/geLPT.png'
} );
```

Regards

**Comment by tophf** [comment]
Your code works here so maybe it's a bug in Chrome e.g. https://crbug.com/825497.

**Reply by nathanrod** [reply]
Quite interesting, i had this issue for months, actually never got it working and i installed violentmonkey today .
